### PR TITLE
Feat rates table

### PR DESCRIPTION
**Pull request by singhsaluja**
### Pull Request
---

#### PR Summary
Briefly describe the changes.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [x] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [ ] Commitizen was used for all commits.
- [ ] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [ ] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

**Comment by singhsaluja** [comment]
@fernandogelin Is there a way to left-align the content in rates.md it's centered right now which looks a little off.

**Comment by fernandogelin** [comment]
@singhsaluja what part are you talking about? Everything looks left aligned to me. Also, can you resolve conflicts and update the branch?

File: themes/gb-theme/layouts/shortcodes/rates.html [hunk 1]
```diff
@@ -1,106 +1,92 @@
-<table style="width: 125%; margin-left:-10%;">
+<table style="width: 120%; margin-left:-10%;">
```
Comment (fernandogelin) [inline themes/gb-theme/layouts/shortcodes/rates.html]: you can play around with width and left margin for the table by changing the style here, not sure if that's what you meant by left align.

File: themes/gb-theme/layouts/shortcodes/rates.html [hunk 2]
```diff
@@ -1,106 +1,92 @@
...
     <tr>
       <td>High End GPU Priority</td>
       <td>16</td>
       <td>256</td>
       <td>4 high-end GPUs (Tesla V100)</td>
-      <td>96</td>
-      <td>Yes</td>
-      <td>$400</td>
       <td>GPU</td>
+      <td>$400</td>
```
Comment (fernandogelin) [inline themes/gb-theme/layouts/shortcodes/rates.html:70]: it's missing a data point here.

**Review by fernandogelin — CHANGES_REQUESTED** [review]

File: themes/gb-theme/layouts/shortcodes/rates.html (hunk 2 above)
Comment (singhsaluja) [reply themes/gb-theme/layouts/shortcodes/rates.html:70]: This is fixed now. Thanks for reminding me.

**Review by singhsaluja — COMMENTED** [review]
@fernandogelin The table looks fine - no align changes required there. But content in markdown is centered. Is there an attribute to set content to be left-aligned?

**Review by mttunca — APPROVED** [review]

**Review by paulstey — APPROVED** [review]

**Pull request merged**